Marketing_Campaign/postgres-docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
       - mypostgres
 
   kensu_run_dbt_postgres:
-    image: vidmasze/kensu-dbt-postgres:dev2
+    image: vidmasze/kensu-dbt-postgres:dev3
```

Comment:
@ivanopagano is it on vidma's repo or kensu?

Reply:
kensu... sorry, made a mistake